src/pages/signatures/getting-started/document-lifecycle.mdx

```diff
@@ -0,0 +1,43 @@
+---
+product: signatures
+category: Getting Started
+sort: 5
+title: Document lifecycle
+subtitle: Learn how Criipto Signatures handles your documents securely and safely.
+---
+import Layout from '../../../layouts/mdx';
+export default Layout;
+
+## Default document lifecycle
+
+- **Signature order is created** when you create a signature order with `createSignatureOrder` you send a long document blobs with it,
+upon receiving it we validate it and then [double-encrypt](#encryption-and-storage) it and store it on Azure servers.
+- **Signatory signs** whenever a signatory needs to sign a document we download and decrypt it to show it to the user,
+once the user signs we embed the signatory evidence (usually e-ID claims returned by Criipto Verify inside the PDF and seal it with Criiptos AATL certificate.
+After sealing the PDF we then again [double-encrypt](#encryption-and-storage) it and store it on Azure servers, ready for the next person to sign.
+- **Signature order is closed** upon closing a signature order we apply a final document timestamp to the PDF to ensure that the data inside is valid for multiple years.
+You should always request document blobs as part of the close response as this is the last time they will be available. After closing the documents will be deleted from Criipto servers.
+
+You can download your document from the Criipto Signatures API at any point **up until after `closeSignatureOrder` is called**.
+
+Auto-expired and cancelled signature orders also have their documents automatically deleted.
+
+## Encryption and storage
+
+Documents are stored in Azure Datacenters in Europe, documents are always double encrypted, both by an Azure-managed key and by a non-Azure, fully EU-owned HSM.
+This protects the documents from any outside access to personal data, in accordance with Screms-II.
+
+## Extended lifecycle
+
+If you wish to extend document lifetime so that it exists on Criiptos servers beyond `closeSignatureOrder` you can send `{retainDocumentsForDays: [number 1-7]}` as input to `closeSignatureOrder`.
+The documents will then stay available on the signature order until the configured amount of days has passed where at which point they will be deleted from Criiptos servers.
+
+To ensure the highest protection of personal data you **must** make sure to clean up signature orders as soon as you can verify you have securely stored documents on your own servers, you can do so by calling the `cleanupSignatureOrder` mutation.
+The `cleanupSignatureOrder` can be called an indefinite amount of times after closing the order and is idempotent.
```

Comment:
`can be called an indefinite amount of times after closing the order and is idempotent` -> `is idempotent. If you are even in a situation where you are uncertain about whether the clean-up mutation succeed, it is better to try it again than not doing anything.`

Someone taking `indefinite` literally is akin to a DoS attack.